packages/node/src/constants.ts

```diff
@@ -0,0 +1,13 @@
+//API calls
+export const API_CALL_TOTAL_TIMEOUT = 29_000;
+export const API_CALL_TIMEOUT = 20_000;
+export const BLOCK_COUNT_HISTORY_LIMIT = 600;
+export const BLOCK_COUNT_IGNORE_LIMIT = 20;
+export const BLOCK_MIN_CONFIRMATIONS = 0;
+export const CONVENIENCE_BATCH_SIZE = 10;
+export const DEFAULT_RETRY_OPERATION_TIMEOUT = 5000;
+export const EVM_PROVIDER_TIMEOUT = 10_000;
+export const GAS_LIMIT = 500_000;
```

Comment:
I think we should drop `GAS_LIMIT` here. It's only used for fulfilling API calls in a single file so I don't think we get much value from extracting it. The generic name will also be really confusing to future readers